src/compiler/diagnosticMessages.json

```diff
@@ -127,11 +127,27 @@
         "category": "Error",
         "code": 1039
     },
+    "'{0}' modifier cannot be used in an ambient context.": {
+        "category": "Error",
+        "code": 1040
+    },
+    "'{0}' modifier cannot be used with a class declaration.": {
+        "category": "Error",
+        "code": 1041
+    },
+    "'{0}' modifier cannot be used here.": {

```

Comment:
When would you use this one? Seems like you could use it anywhere. Also, will the user understand why they can't use something 'here'?